create-router/index.js

```diff
@@ -85,13 +85,15 @@ export function createRouter(routes) {
     }
   })
 
+  let setState = router.set
+  delete router.set
```

Comment:
We can wrap delete statement with non production env check. It will save few bytes.
Current version add 11 B, with check it would be only 4. Not sure if it makes sense or not.
@ai